src/git-operations.ts

```diff
@@ -9,6 +9,47 @@ const HEAD = 'HEAD';
 type DiffMap = Map<string, string[]>;
 const DIFFS: DiffMap = new Map();
 
+/**
+ * Gets the HTTPS URL of the current GitHub remote repository. Assumes that
+ * the git config remote.origin.url string matches one of:
+ *
+ * - https://github.com/OrganizationName/RepositoryName
+ * - git@github.com:OrganizationName/RepositoryName.git
+ *
+ * If the URL of the "origin" remote matches neither pattern, an error is
+ * thrown.
+ *
+ * @returns The HTTPS URL of the repository, e.g.
+ * https://github.com/OrganizationName/RepositoryName
+ */
+export async function getRepositoryHttpsUrl(): Promise<string> {
```

Comment:
Thoughts on getting this from `package.json` instead? That's what `auto-changelog` does by default.

Getting it from the manifest has the benefit of this working correctly for forks as well, where a contributor would want the canonical repo URL in `package.json` to be used rather than their fork's URL.

Reply:
Right, fair enough. This seems like a good approach then.